src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1397,6 +1412,11 @@ private TextLoader(IHost host, ModelLoadContext ctx)
             if (_separators.Contains(':'))
                 host.CheckDecode((_flags & OptionFlags.AllowSparse) == 0);
 
+            if (ctx.Header.ModelVerWritten >= 0x0001000D)
+            {
+                _decimalMarker = ctx.Reader.ReadChar();
+                host.CheckDecode(_decimalMarker == '.' || _decimalMarker == ',');
+            }
```

Comment:
BTW, in ML.NET we don't provide any public API to serialize or de-serialize TextLoader. This is done through MAML, and perhaps it's also done through NimbusML (although I haven't confirmed this).

So to actually test that the TextLoader is being correctly saved to disk and loaded back, I found that to test this we can add a TestCommandBase test such as this one, to actually test serialization:
https://github.com/dotnet/machinelearning/pull/5125/files#diff-67ab0faabe9ac8db40f9ccb7fad4fafd

Reply:
Ok, now that my previous pr #5147 has been merged, there's no need to bump up the VersionInfo here, nor to test backward compataibility, or serialization-deserialization... as this is already being teste on my latest PR. 👍